trace-archiver-hdf5/src/main.rs

```diff
@@ -99,22 +77,13 @@ async fn main() -> Result<()> {
     .set("enable.auto.commit", "false")
     .create()?;
 
-    let topics_to_subscribe: Vec<String> = vec![args.event_topic, args.trace_topic]
-        .into_iter()
-        .flatten()
-        .collect();
-    let topics_to_subscribe: Vec<&str> = topics_to_subscribe.iter().map(|i| i.as_ref()).collect();
-    if topics_to_subscribe.is_empty() {
+    if let Some(trace_topic) = args.trace_topic {
```

Comment:
Trace topic should be made to be required now.

Reply:
Should the trace file also be required?

Reply:
Yes.